index.html

```diff
@@ -9,6 +9,11 @@
 			href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600;700;800;900&display=swap"
 			rel="stylesheet"
 		/>
+
+		<link
```

Comment:
Let's use imports for fonts in `index.css` instead of links in the `index.html`

- Change this for both Raleway and Poppins

Reply:
Resolved in most recent PR